src/libYARP_dev/src/Drivers.cpp

```diff
@@ -87,7 +87,10 @@ class DriversHelper : public YarpPluginSelector {
             settings.readFromSearchable(prop,name);
             settings.open(lib);
             ConstString location = lib.getName().c_str();
-            if (location=="") continue;
+            if (location=="") {
```

Comment:
I tried by putting a wrong library name and it enters the if. So yes, wrong library name gives an empty location. I'll add a comment.